src/material-examples/button-types/button-types-example.html

```diff
@@ -1,17 +1,50 @@
-<div class="example-container">
-  <h3>Normal Buttons</h3>
-  <div class="button-row">
-    <button md-button>Flat button</button>
-    <button md-raised-button>Raised button</button>
-    <button md-fab><md-icon>check</md-icon></button>
-    <button md-mini-fab><md-icon>check</md-icon></button>
-  </div>
+<h3>Basic Buttons</h3>
+<div class="button-row">
+  <button md-button>Basic</button>
+  <button md-button color="primary">Primary</button>
+  <button md-button color="accent">Accent</button>
+  <button md-button color="warn">Warn</button>
+  <button md-button disabled>Disabled</button>
+  <a md-button routerLink=".">Link</a>
+</div>
+
+<h3>Raised Buttons</h3>
+<div class="button-row">
+  <button md-raised-button>Basic</button>
+  <button md-raised-button color="primary">Primary</button>
+  <button md-raised-button color="accent">Accent</button>
+  <button md-raised-button color="warn">Warn</button>
+  <button md-raised-button disabled>Disabled</button>
+  <a md-raised-button routerLink=".">Link</a>
+</div>
+
+<h3>Icon Buttons</h3>
+<div class="button-row">
+  <button md-icon-button><md-icon class="md-24" aria-label="favorite">favorite</md-icon></button>
+  <button md-icon-button color="primary"><md-icon class="md-24" aria-label="favorite">favorite</md-icon></button>
```

Comment:
Should be a bit more specific w/ the aria-label:
`"Example icon-button with a heart icon"`